CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [unreleased]
 
+### Added
+- Added additional example called `croncat.rs` to showcase more in depth `fast_forward` feature: https://github.com/near/workspaces-rs/pull/126
```

Comment:
note this does not change the API of the crate so this might not be something you want to include in the changelog

Reply:
huh, so normally we don't want examples inside changelog? It make sense for internals being reworked around to not make it in the changelog, but I thought since example was public, it would be worth noting

Reply:
I mean, the point of the changelog is to see what has changed when deciding to update or see how the library was changed, but you can create whatever semantics if you think it would be beneficial. I don't think there is a rule against something like this, but I just can't intuitively see the benefit here.

cc @miraclx @itegulov do you have any thoughts about if example changes should be included in our changelogs to keep things consistent?

Reply:
I agree with Austin here. Don't think examples should be included in the changelog, I view them as a separate entity from the library itself.

Reply:
hmm I see, I'll just revert then